README.md

```diff
@@ -10,22 +10,28 @@ products:
 
 ## Getting the Driver
 
-Binary versions of the driver, available for multiple operating systems and
-multiple architectures, can be obtained from our [download server]. The
-source code is made available via [GitHub].
-
-Packages for the driver's dependencies, libuv (1.x), OpenSSL, and zlib are also
-provided under the `dependencies` directory for each platform (if applicable).
-__Note__: CentOS and Ubuntu use the version of OpenSSL and zlib provided with the
-distribution:
-
-* [CentOS 6][centos-6-dependencies]
-* [CentOS 7][centos-7-dependencies]
-* [CentOS 8][centos-8-dependencies]
-* [Ubuntu 14.04][ubuntu-14-04-dependencies]
-* [Ubuntu 16.04][ubuntu-16-04-dependencies]
-* [Ubuntu 18.04][ubuntu-18-04-dependencies]
-* [Windows][windows-dependencies]
```

Comment:
Re-worked this section (and removed the links) since there is no common location in Artifactory that can be used as an entry point for dependencies.  We can point users at the "builds" directory but underneath there we have subdirectories for version number and build number.. so it seemed better to provide guidance on how to find the deps rather than try to come up with some arbitrary URL scheme to point at them.